### DISCUSSION TITLE
Devbox is now using this project!

### DISCUSSION THREAD

**Discussion by thenonameguy** [F1bonacc1/process-compose#50] [Show and tell]
See the blog-post:
https://www.jetpack.io/blog/devbox-0-4-7/

Congrats @F1bonacc1 on making a really useful software that's now integrated in 2 Nix based developer setup tools (https://devenv.sh being the first) :tada: :clap:

**Comment by F1bonacc1** [comment]
That's cool @thenonameguy!

That wouldn't be possible without you introducing this project to the Nix community.
Thank you for that and for all the work you are doing for the community behind the scenes! 

🙏🏼🙏🏼🙏🏼
